osvr/RenderKit/RenderManagerOpenGL.cpp

```diff
@@ -457,6 +466,18 @@ namespace renderkit {
         delete m_library.OpenGL;
     }
 
+    bool RenderManagerOpenGL::GetTimingInfo(size_t whichEye, RenderTimingInfo& info) {
+      if(m_toolkit.getRenderTimingInfo) {
```

Comment:
[Style] I'd probably invert this and write `if (!m_toolkit.getRenderTimingInfo) return false;` and then the remainder of the function wouldn't need to be indented an extra level. It doesn't harm readability in this small of a function, but can really help in longer functions.